packages/main/src/Tokenizer.ts

```diff
@@ -286,14 +286,14 @@ class Tokenizer extends UI5Element {
 		}
 	}
 
-	_tokenKeyboardDelete(e: CustomEvent<TokenDeleteEventDetail>, token: Token) {
+	_tokenKeyboardDelete(e: CustomEvent<TokenDeleteEventDetail> | { detail?: { backSpace: boolean }, target: EventTarget }, token: Token) {
```

Comment:
This signals not the based code design
`e: CustomEvent<TokenDeleteEventDetail> | { detail?: { backSpace: boolean }, target: EventTarget }`

Refactor the TokenDeleteEventDetail, or even better new method for the MultiCombox to call with a nice looking  signature, parameter